test/common/network/proxy_protocol_test.cc

```diff
@@ -53,7 +54,11 @@ class ProxyProtocolTest : public testing::Test {
   std::shared_ptr<MockReadFilter> read_filter_;
 };
 
-TEST_F(ProxyProtocolTest, Basic) {
+// Parameterize the listener socket address version.
+INSTANTIATE_TEST_CASE_P(IpVersions, ProxyProtocolTest,
+                        testing::ValuesIn(TestEnvironment::getIpVersionsForTest()));
```

Comment:
socket_ in the test constructor binds which fails in IPv4/IPv6 only environments. It's correct that the test logic doesn't benefit from parameterization.